nexus/reconfigurator/planning/src/blueprint_builder/clickhouse.rs

```diff
@@ -143,9 +137,27 @@ impl ClickhouseAllocator {
         // If we fail to retrieve any inventory for keepers in the current
         // collection than we must not modify our keeper config, as we don't
         // know whether a configuration is ongoing or not.
+        //
+        // There is an exception to this rule: on *new* clusters that have
```

Comment:
I had an epiphany here. When we start from 0 nodes, we can actually go straight to the desired configuration. There is no existing configuration and therefore this is not a reconfiguration requiring only adding or removing a single node. 

This will make bringing up new clusters via OMDB on existing systems much quicker and I think negates the need to deal with such complexity in RSS.

Reply:
Implemented in [f73ea10](https://github.com/oxidecomputer/omicron/pull/6627/commits/f73ea1083f9a3de0014ba903b96bad8492db35a1)